packages/usa-in-page-navigation/src/index.js

```diff
@@ -42,23 +44,35 @@ const setActive = (el) => {
 };
 
 /**
- * Return an array of all visible h2 and h3 headings from the designated main content region.
- * These will be added to the component link list.
+ * Return an array of visible headings from the designated content region.
+ * These items will be added to the component link list.
  *
- * @param {HTMLElement} mainContentSelector The designated main content region
+ * @param {HTMLElement} selectedContentRegion The content region the component should pull headers from
+ * @param {String} selectedHeadingTypes The list of heading types that should be included in the link list
  *
  * @return {Array} - An array of visible headings from the designated content region
  */
-const getSectionHeadings = (mainContentSelector) => {
-  const sectionHeadings = document.querySelectorAll(
-    `${mainContentSelector} h2, ${mainContentSelector} h3`
-  );
+const getSectionHeadings = (selectedContentRegion, selectedHeadingTypes) => {
+  const headingTypes = selectedHeadingTypes.indexOf(" ")
+    ? selectedHeadingTypes.split(" ")
+    : selectedHeadingTypes;
+  const contentHeadingTypes = [];
```

Comment:
Alternative approach to looping and pushing headings would be to query them. Simpler, but unsure if I'm missing something from the current approach.

```js
  // Ideally it'd be a root element for `region`.
  const region = document.querySelector(selectedContentRegion);
  const sectionHeadings = region.querySelectorAll(headingTypes);
```

Reply:
Made some updates in [30a3b9e](https://github.com/uswds/uswds/pull/5444/commits/30a3b9e68787790f8b133a194fd3312b52cd5682). Let me know if you were looking for anything different.